Makefile

```diff
@@ -67,7 +67,7 @@ test-cov: ##@tests Runs all tests with coverage
 
 .PHONY: test-int
 test-int: ##@tests Runs integration tests
-	poetry run pytest  -m "integration" -vv -l -x
+	poetry run pytest ./tests_integration -m "integration" -vv -l -x
```

Comment:
This is to prevent pytest from loading the `apps.handers` module when collecting test files. That would prevent us from patching the envvars for kopf timers like [this one](https://github.com/Twingate/kubernetes-operator/blob/37873f05e5997d3824e6ea4acdeb44c5f4202627/app/handlers/handlers_resource.py#L94-L96).